File: Learning Android App/firebase/Chats/app/src/main/java/com/example/chats/MainActivity.java
package com.example.chats;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.LayoutInflater;

import com.example.chats.Adapters.FragmentsAdapter;
import com.example.chats.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
//here in this project we will do the work of viewpager
//which will help to see the chats section and status section and calls section


    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.viewPager.setAdapter(new FragmentsAdapter(getSupportFragmentManager()));
        binding.tab.setupWithViewPager(binding.viewPager);

    }
}

File: Learning Android App/web3Java/app/src/main/java/com/example/web3java/MainActivity.kt
package com.example.web3java

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.web3java.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val web3 = Helper()

        binding.balance.setOnClickListener {
            binding.textView.text = web3.getBalance("0x501e0636c64b28840e0C38409Beb87d6BdfA835A")
        }

    }
}